packages/css-library/src/utilities/width.scss

```diff
@@ -0,0 +1,38 @@
+@use 'uswds-core/src/styles/functions' as *;
+@use '../tokens/width' as *;
+@use './settings' as *;
+
+$u-width-base: (
+  width: (
+    base: 'vads-width',
+    modifiers: null,
+    values: map-collect($tokens-width),
+    settings: $width-settings,
+    property: 'width',
+    type: 'utility',
+  ),
+);
+
+$u-min-width: (
+  min-width: (
+    base: 'vads-min-width',
```

Comment:
These are different from the USWDS pattern of `minw-*` but I think our approach is better from a readability standpoint.